packages/shaders/src/shader-mount.ts

```diff
@@ -50,25 +50,30 @@ export class ShaderMount {
     /** The maximum resolution (on the larger axis) that we render for the shader. Use virtual pixels, will be multiplied by display DPI to get the actual resolution. Actual CSS size of the canvas can be larger, it will just lose quality after this */
     maxResolution = 1920
   ) {
-    // Create the canvas element and mount it into the provided div
-    const canvas = document.createElement('canvas');
-    mountToDiv.style.contain = 'strict';
-    mountToDiv.style.position = 'relative';
-    canvas.style.position = 'absolute';
-    canvas.style.inset = '0';
-    canvas.style.zIndex = '-1';
-    canvas.style.width = '100%';
-    canvas.style.height = '100%';
-    this.canvas = canvas;
-    this.mountToDiv = mountToDiv as PaperShaderDivWrapperElement;
-    mountToDiv.appendChild(canvas);
+    if (parentElement instanceof HTMLElement) {
+      this.parentElement = parentElement as PaperShaderElement;
+    } else {
+      throw new Error('Parent element must be an HTMLElement');
+    }
+
+    if (!document.querySelector('style[data-paper-shaders]')) {
+      const styleElement = document.createElement('style');
+      styleElement.innerHTML = defaultStyle;
+      styleElement.setAttribute('data-paper-shaders', '');
+      document.head.prepend(styleElement);
+    }
+
+    // Create the canvas element and mount it into the provided element
+    const canvasElement = document.createElement('canvas');
+    this.canvasElement = canvasElement;
+    this.parentElement.prepend(canvasElement);
```

Comment:
Prepend the canvas so that children are always on top (even if they have z -1) and so that its order in dev tools matches the visual order